### ISSUE TITLE
deprecation warnings ?

### ISSUE THREAD

**Issue by srob1** [secynic/ipwhois#203]
I am using IPWhois on a raspberry pi under the Python 2.7.9 Shell and am getting deprecation warnings about allow_permutations and _parse_fields_dns() when running the lookup_rdap example from the examples page:

Warning (from warnings module):
  File "/usr/local/lib/python2.7/dist-packages/ipwhois/net.py", line 138
    warn('allow_permutations has been deprecated and will be removed. '
UserWarning: allow_permutations has been deprecated and will be removed. It is no longer needed, due to the deprecation of asn_alts, and the addition of the asn_methods argument.

Warning (from warnings module):
  File "/usr/local/lib/python2.7/dist-packages/ipwhois/asn.py", line 178
    warn('IPASN._parse_fields_dns() has been deprecated and will be '
UserWarning: IPASN._parse_fields_dns() has been deprecated and will be removed. You should now use IPASN.parse_fields_dns().

The warnings resulted from executing this code which pretty much comes straight from the RDAP basis usage example:

obj = IPWhois('74.125.225.229');
results = obj.lookup_rdap(depth=1);
print results['asn_cidr'];

**Comment by secynic** [comment]
The core library is still calling these. I'll get it fixed next version. Nothing to worry about as far as functionality.

**Comment by pmuller** [comment]
Same issue here. Reading the code it appears that's the way the library is designed. When calling ``lookup()``, internals are made toward ``_parse_fields_*()`` functions. No way to bypass this.

Is there an ETA for a fix release?

Thanks :)

**Comment by secynic** [comment]
Will work this one once the dedup branch is merged to dev (soon). I also just realized it is using UserWarning and not DeprecationWarning.

You can suppress these in the meantime:
```python
import warnings
with warnings.catch_warnings():
    warnings.filterwarnings("ignore", category=UserWarning)
    # put your lookup call here
```

**Comment by brettus78** [comment]
I still get this issue. Has it actually been resolved?
PS. I am using the suppression method for now - but this is probably not ideal?

**Comment by WalterMccan** [comment]
This is still not fixed. The temp solution works but it does introduce a little bloat to the code.

**Comment by joneswac** [comment]
Using temp solution is working, just posting to keep tabs on this issue.

**Comment by secynic** [comment]
Merged to dev in #224
